src/components/Web3Status/index.tsx

```diff
@@ -107,7 +108,7 @@ const Web3StatusConnected = styled(Web3StatusGeneric)<{
 const AddressAndChevronContainer = styled.div`
   display: flex;
 
-  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.navSearchInputVisible}px`}) {
+  @media only screen and (max-width: ${() => `${navSearchInputVisibleSize}px`}) {
```

Comment:
this can just be static now and not a function right?

Reply:
Good catch, going to take a bit to get used to styled-components / vanilla extract.